File: server/src/game/tools/conf/output/ts/QuotesTreeConfig.ts
class QuotesTreeConfig {
    private static _dic:Dictionary<QuotesTreeConfig> = new Dictionary<QuotesTreeConfig>();
    private static _arr:Array<QuotesTreeConfig> = new Array<QuotesTreeConfig>();

	 public Id :number ;
     public ParentId :number ;
     public Name :string ;
     public TextId :number ;
     public Type :number ;
     public ListId :number ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<QuotesTreeConfig>{
        return  QuotesTreeConfig._dic;
    }
    public static get Arr():Array<QuotesTreeConfig>{
        return QuotesTreeConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:QuotesTreeConfig = <QuotesTreeConfig>json[i];
            QuotesTreeConfig._arr.push(config);
            QuotesTreeConfig._dic.add(config.Id,config);
        }
    }
}

File: server/src/game/tools/conf/output/go/QuotesTreeConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type QuotesTreeConfigElem struct {
	Id int
	ParentId int
	Name string
	TextId int
	Type int
	ListId int
	
}

type QuotesTreeConfig0 struct {
	Dic map[int]*QuotesTreeConfigElem
	Arr []*QuotesTreeConfigElem
}

var (
	QuotesTreeConfig QuotesTreeConfig0
)

func loadQuotesTreeConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	QuotesTreeConfig.Dic = make(map[int]*QuotesTreeConfigElem)
	QuotesTreeConfig.Arr = make([]*QuotesTreeConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &QuotesTreeConfig.Arr)
	for _,conf := range QuotesTreeConfig.Arr{
		QuotesTreeConfig.Dic[conf.Id] = conf
	}
	
}
